### PR TITLE
Revert "Docs 5684 toolbar updates mc"

### PR DESCRIPTION
**Pull request by mcherem**
Reverts mulesoft/docs-design-center#96

### CODE REVIEW

File: modules/ROOT/pages/fd-publish-app-as-template.adoc [hunk 1]
```diff
@@ -33,9 +33,9 @@ Templates do not include the authentication credentials that are specified for t
 
 == Procedure
 
-. Select *Publish Template* from the drop-down menu which is in the top-right corner of the Flow Designer screen.
+. Click the *Publish* button, which is in the top-right corner of the Flow Designer screen.
```
Comment (dhirsch1) [inline modules/ROOT/pages/fd-publish-app-as-template.adoc]: Click *Publish*  from...

File: modules/ROOT/pages/import-template.adoc [hunk 2]
```diff
@@ -36,7 +36,7 @@ For example, if a flow has a card for the Slack connector and posts a message to
 +
 For guidance while you are customizing, refer to the *Notes* section in each card.
 . When you are ready, click *Run* to run the application temporarily and test it. While an application is running, you can trigger its flows and verify that data moves through the flows correctly. After you click *Run*, the application continues to run for 60 hours unless you make changes to it. Whenever you make a change, the application stops running.
-. When you are satisfied with the application, select *Deploy Application* under the drop-down menu to run it in a sandbox environment or in a production environment.
```
Comment (dhirsch1) [inline modules/ROOT/pages/import-template.adoc]: Remove "in"

File: modules/ROOT/pages/promote-app-prod-env-design-center.adoc [hunk 3]
```diff
@@ -24,7 +24,7 @@ image::deploying-your-project-9cee7.png[]
 
 == What to do next
 
-You can go to Runtime Manager to manage the application. To get there from Flow Designer, click the three-dots icon to the right of the drop-down menu and select *View in Runtime Manager*.
+You can go to Runtime Manager to manage the application. To get there from Flow Designer, click the three-dots icon to the left of the *Deploy* button and select *View in Runtime Manager*.
```
Comment (dhirsch1) [inline modules/ROOT/pages/promote-app-prod-env-design-center.adoc]: "Access Runtime Manager to..."

File: modules/ROOT/pages/troubleshooting-reference.adoc [hunk 4]
```diff
@@ -8,7 +8,7 @@ Error messages can appear in two places in Flow Designer:
 * The bottom-right corner of your screen displays error messages in red boxes.
 +
 When you see an error message in this location and you are unable to resolve the error, open a support case for Flow Designer and paste the content of the message into the support case.
-* The top-right corner of your screen displays them next to the *Test* button and the drop-down menu.
+* The top-right corner of your screen displays them between the *Run* and *Deploy* buttons.
```
Comment (dhirsch1) [inline modules/ROOT/pages/troubleshooting-reference.adoc]: Change "them" to "the error messages"

File: modules/ROOT/pages/fd-tour.adoc [hunk 5]
```diff
@@ -35,10 +35,10 @@ What are data types?:: Data types show the formats in which data are used in a f
...
 . The toolbar
 +
-Here you can undo and redo changes in a project, download a project, change the zoom level of the canvas, run the application to test it, or deploy the application to another environment.
-// . The control bar
-// +
-// Here you can run the application for testing, as well as deploy the application to another environment.
+Here you can undo and redo changes in a project, download a project, and change the zoom level of the canvas.
+. The control bar
++
+Here you can run the application for testing, as well as deploy the application to another environment.
```
Comment (dhirsch1) [inline modules/ROOT/pages/fd-tour.adoc]: I dislike the text "Here you can," but I have seen it all over the place.

File: modules/ROOT/pages/fd-publish-app-as-template.adoc (hunk 1 above)
Comment (mcherem) [reply modules/ROOT/pages/fd-publish-app-as-template.adoc]: Should confirm in the style guide moving forward.

File: modules/ROOT/pages/fd-tour.adoc (hunk 5 above)
Comment (mcherem) [reply modules/ROOT/pages/fd-tour.adoc]: Okay, thanks. It's a style.

File: modules/ROOT/pages/import-template.adoc (hunk 2 above)
Comment (mcherem) [reply modules/ROOT/pages/import-template.adoc]: okay, but not this time around.

File: modules/ROOT/pages/promote-app-prod-env-design-center.adoc (hunk 3 above)
Comment (mcherem) [reply modules/ROOT/pages/promote-app-prod-env-design-center.adoc]: STET

**Pull request merged**